src/Umbraco.Core/Services/AuditService.cs

```diff
@@ -18,11 +26,15 @@ public sealed class AuditService : RepositoryService, IAuditService
         ILoggerFactory loggerFactory,
         IEventMessagesFactory eventMessagesFactory,
         IAuditRepository auditRepository,
-        IAuditEntryRepository auditEntryRepository)
+        IAuditEntryRepository auditEntryRepository,
+        IUserService userService,
+        IEntityRepository entityRepository)
```

Comment:
I think we should either obsolete the old constructor and add a new one, or go back and add the new constructor and obsolete this one in V11

Reply:
Yep done here: https://github.com/umbraco/Umbraco-CMS/pull/13875